github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -2307,6 +2307,8 @@ presubmits:
         env:
         - name: TARGET
           value: k8s-1.26-centos9-sig-operator-upgrade
+        - name: KUBEVIRT_PSA
+          value: "true"
```

Comment:
Upgrade lane should not have PSA enabled, since it requires previous version to be PSA compliant, which is not

Reply:
+1

Reply:
Do we have documentation about what people need to do in order to enable PSA on their systems after the upgrade?

Reply:
Or is the process as straight forward as: we can upgrade and then enable PSA?

Reply:
There is Kubernetes documentation. From Kubevirt side people need to be aware that only 0.59 is going to support it. This should be in the release note afaik. No additional configuration on Kubevirt side is needed.

Reply:
I tend to disagree here, since release-0.59 should be PSA ready, no? So the change and split for operator where update is PSA disabled should be done in `presubmits-0.59.yaml`.

Seel #2611 where I am going to apply that

Reply:
The exact point of these changes for operator lanes would be that we see whether we can upgrade from release-0.59 to main with PSA enabled, which should be the desired outcome. And that is what we want to test here.

Reply:
Yes, but the tests install the previous version (0.58.1 now) which is not PSA compliant

Reply:
>The exact point of these changes for operator lanes would be that we see whether we can upgrade from release-0.59 to main with PSA enabled, which should be the desired outcome. And that is what we want to test here.

I agree but we are testing the main against 0.58 releases because there is no official release of 0.59. (Therefore we need the split)I meant to raise this in our meeting.

Reply:
@xpivarc Should we consider the idea of delete this line?
https://github.com/kubevirt/kubevirt/blob/bed79f5a92e46210474334e8b4d729225f85fbd5/tests/operator/operator.go#L3163

Reply:
We need a better process overall. I would push this out for now and we can do a retrospective on the 59 release.

Reply:
So I understand that this is kind of chicken and egg here? Would the upgrade work with a 0.59 RC also?

Reply:
`upgrade with psa enabled` should work with 0.59-rc0 (since it is psa compliant).
I am doing a local test right now to validate it.
But we will have the same problem in release-0.59 branch; and there we cannot consider a real upgrade if we go from release-0.59 to release-0.59

Reply:
Yes, understood. I'll be working on changing the lanes for release-0.59 today, where we are splitting the operator lanes in upgrade and config where upgrade will be run with `KUBEVIRT_PSA=false`

Reply:
Thanks! 

@xpivarc we can also add `PREVIOUS_RELEASE_TAG` env variable of the job, instead of changing the code. WDYT?

Reply:
> I am doing a local test right now to validate it.

Confirmed

Reply:
> Thanks!
> 
> @xpivarc we can also add `PREVIOUS_RELEASE_TAG` env variable of the job, instead of changing the code. WDYT?

Where? The only problem I see is that someone needs to make sure it will stay correct(e.g new release).

Reply:
It can be passed as environment variable of the job AFAIU
```
        - name: PREVIOUS_RELEASE_TAG
        - value: v0.59.0-rc.0
```

> The only problem I see is that someone needs to make sure it will stay correct(e.g new release).
Sure! This should be only a temporary fix and must be removed once release-0.59 is officially released.